Arduino-Bling/RobotBling/animations.ino

```diff
@@ -443,3 +443,60 @@ void flash(unsigned long timeInterval, uint32_t color, uint8_t brightness)
   setAllPixelsOff();
   showPixels();
 }
+
+void rainbowCycle(int SpeedDelay) {
+  byte *c;
+  uint16_t i, j;
+
+  for(j=0; j<256*5; j++) { // 5 cycles of all colors on wheel
```

Comment:
Please reformat the code to match the existing coding style, i.e. placement of `{}`